core/src/main/java/com/linecorp/armeria/server/ServiceRequestContext.java

```diff
@@ -258,6 +260,28 @@ default String pathParam(String name) {
         return pathParams().get(name);
     }
 
+    /**
+     * Returns the query parameters mapped by the {@link Route} associated with the {@link Service}
+     * that is handling the current {@link Request}.
+     */
+    QueryParams queryParams();
+
+    /**
+     * Returns the values of the specified query parameter.
+     */
+    @Nullable
+    default List<String> queryParams(String name) {
+        return queryParams().getAll(name);
+    }
+
+    /**
+     * Returns the value of the specified query parameter.
```

Comment:
ditto. It's hard to notice the difference between this description and the description of the above method. 😉